ragen/env/deepcoder/utils.py

```
@@ -0,0 +1,209 @@
+import json
+import os
+import subprocess
+import textwrap
+from typing import Optional, Tuple
+
+from datasets import concatenate_datasets, load_dataset
+
+try:
+    from rllm.data.dataset import DatasetRegistry
+except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)